tdrs-backend/docs/user_role_management.md

```diff
@@ -0,0 +1,168 @@
+# User Role Management
+
+## Updating User Roles through the Django Admin Interface
+
+**Description**
+
+For the OFA MVP we will be assigning and updating application user roles through the
+Django Admin Interface. This can be accessed via the backend at:
+
+`<backend-domain-name>/admin`
+
+The admin interface requires special privileges which can only be granted via the
+Django CLI [Detailed Below](#cli). 
+
+Once a user has been granted privileges they can go to the admin page described above
+and log in. The admin interface provides links to Users, STTs, Regions and Groups which
+can can be each be modified through the interface.
+
+### Log in to Admin
+
+- Enter your username and password to gain access
```

Comment:
@carltonsmith A couple of questions about this:

+ This will be different from the user's Login.gov password, is that correct? How does this interact with the `user.set_unusable_password()` call made from `login.py`?
+ This isn't protected by 2FA in the proposed PR, is that correct?

Reply:
- Yes, this will be different from the user's login.gov password. It doesn't interfere with the `user.set_unusable_password()` call. That call is necessary for Django to create the account, but doesn't break when the user's password is set for Django Admin.

- That's correct. I don't know if there is a way to turn on MFA for the admin interface but I can look into it.

Reply:
Noting that we broke this out into a separate issue: https://app.zenhub.com/workspaces/tdrs-product-backlog-5f2c6cdc7c0bb1001bdc43a5/issues/raft-tech/tanf-app/517